tox.ini

```diff
@@ -2,7 +2,7 @@
 addopts=--tb=short
 
 [tox]
-envlist = py27,py35,isort-check,isort-fix,lint,docs
+envlist = isort-check,isort-fix,lint,py27,py35,py36,docs
```

Comment:
By changing the order the tox-build might fail faster on simple issues like isort/linting. Otherwise you have to wait until all you test cases to run only to find out you missed a proper import sorting.